### PR TITLE
Minor updates to how-to guide on gateways 

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/community/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### Summary
🔎🚀🛠️

Updated the gateway tutorial and the installation guide in the docs. The gateway tutorial now has a more consistent and clear example code and instructions. The installation guide now includes a step to create and set a Kubernetes cluster.

> _Sing, O Muse, of the skillful coder who changed the tutorial_
> _To make it easier for the learners of the azure container_
> _Who first renamed the section and the file of app.bicep_
> _And then added a new step for the cluster of Kubernetes_

### Walkthrough
*  Simplify the gateway tutorial by renaming and updating the first step ([link](https://github.com/radius-project/docs/pull/851/files?diff=unified&w=0#diff-5a42e650d2f8fc8f765d9418143248e29590f0ee42220d27ed99417e6f1c1e73L19-R21))
*  Rename the container resource and the gateway rule destination to match the image name ([link](https://github.com/radius-project/docs/pull/851/files?diff=unified&w=0#diff-f514c6bf6a78ae5077ea611cd856ccc68bd719db2aba506197b56f2072b4dd3fL7-R8), [link](https://github.com/radius-project/docs/pull/851/files?diff=unified&w=0#diff-f514c6bf6a78ae5077ea611cd856ccc68bd719db2aba506197b56f2072b4dd3fL31-R31))
*  Add a new step to create and set a Kubernetes cluster before installing Radius ([link](https://github.com/radius-project/docs/pull/851/files?diff=unified&w=0#diff-1f0740531db22f0542ce7b48e211f8438fa124b0e82019dd41beaeab5f697527R15-R25))

### CODE REVIEW

File: docs/shared-content/installation/install-radius/initialize-radius.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ Initializing Radius installs the Radius control-plane and creates a Radius Envir
    kubectl config current-context
    ```
 
+1. Create a Kubernetes cluster
+
+   Radius runs inside [Kubernetes]({{< ref "guides/operations/kubernetes" >}}). Create one from the [supported k8s clusters]({{< ref "/guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}})
```
Comment (jasonviviano) [inline docs/shared-content/installation/install-radius/initialize-radius.md]: @Reshrahim Hey it seems you've duplicated the step above again by mistake or maybe forgot to delete the one above.

Comment (Reshrahim) [reply docs/shared-content/installation/install-radius/initialize-radius.md]: thanks for catching that.

File: docs/content/guides/author-apps/networking/howto-gateways/index.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ This guide will walk you through how to setup a gateway for routing internet tra
 - [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-radius-bicep-vs-code-extension" >}})
 - [Radius environment]({{< ref "installation#step-3-initialize-the-radius-control-plane-and-the-radius-environment" >}})
 
-## Step 1: Define a container
+## Step 1: View the container
```
Comment (AaronCrawfis) [inline docs/content/guides/author-apps/networking/howto-gateways/index.md:19]: hmm without explicitly telling the user to run `rad init` and select "yes" to scaffold an application I think "Define a container" works better, along with "in a new file".

Comment (Reshrahim) [reply docs/content/guides/author-apps/networking/howto-gateways/index.md:19]: The pre-requisite step Radius environment does cover the step to select yes and set up the application. Either we go with that or make env creation a separate step here

Comment (Reshrahim) [reply docs/content/guides/author-apps/networking/howto-gateways/index.md:19]: closing as we decided to make how-tos standalone and not operate with the assumption of user scaffolding an application via rad init

**Pull request closed without merging**